### PR TITLE
Added Spotlight instructions for Mac users

### PR DESCRIPTION
**Pull request by atkinsdavid**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
 - [x] The title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
 - [x] If the PR is related to an open issue, use a [relevant keyword](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) and reference it with the `#` sign and the issue number.
 - [x] You have previewed your markdown (if any) in the [Odin Markdown Preview Tool](https://www.theodinproject.com/lessons/preview)
 - [x] If changes are requested, please make the changes in a timely manner. After you submit the changes, request another review from the maintainer (top right).

#### 1. Describe the changes made and include why they are necessary or important:

Added instructions for Mac users to use Spotlight search to open Terminal. I felt like this was very inline with the idea further down the page, "Programmers are lazy."

#### 2. Related Issue

N/A

### CODE REVIEW

File: foundations/installations/command_line_basics.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ That black screen or window is the command line interface (CLI), where you're ab
 In this introductory lesson to the command line, you'll learn how to navigate around your computer and how to manipulate files and directories (also known as folders) directly from the comfort of the command line. You'll soon see that this isn't as difficult as you may think. The commands you will learn in this lesson are very straightforward, so don't be intimidated by the prospect of using the command line for the first time.
 
 #### Test Drive Your Terminal
-Open a terminal on your computer.
-- **Linux**: open the programs menu and search for "Terminal". You can also open the terminal by pressing `CTRL + ALT + T` on your keyboard.
-- **MacOS**: Open your Applications > Utilities folder and find "Terminal".
+Open a terminal on your computer.  
+- **Linux**: open the programs menu and search for "Terminal". You can also open the terminal by pressing `CTRL + ALT + T` on your keyboard.
```
Comment (I3uckwheat) [inline foundations/installations/command_line_basics.md:11]: Hey, can we remove these additional whitespaces :D

**Review by I3uckwheat — CHANGES_REQUESTED** [review]

File: foundations/installations/command_line_basics.md [hunk 2]
```diff
@@ -7,9 +7,9 @@ That black screen or window is the command line interface (CLI), where you're ab
 In this introductory lesson to the command line, you'll learn how to navigate around your computer and how to manipulate files and directories (also known as folders) directly from the comfort of the command line. You'll soon see that this isn't as difficult as you may think. The commands you will learn in this lesson are very straightforward, so don't be intimidated by the prospect of using the command line for the first time.
 
 #### Test Drive Your Terminal
-Open a terminal on your computer.
-- **Linux**: open the programs menu and search for "Terminal". You can also open the terminal by pressing `CTRL + ALT + T` on your keyboard.
-- **MacOS**: Open your Applications > Utilities folder and find "Terminal".
+Open a terminal on your computer.  
+- **Linux**: open the programs menu and search for "Terminal". You can also open the terminal by pressing `CTRL + ALT + T` on your keyboard.  
+- **MacOS**: Open your Applications > Utilities folder and find "Terminal". You can also use Mac's Spotlight search to open Terminal. Press 'CMD + SPACE' to open Spotlight, and search for "Terminal" and press 'RETURN' to open it.
```
Comment (xandora) [inline foundations/installations/command_line_basics.md:12]: We can drop this bit and assume it's a macOS feature givin we're in the macOS specific instructions. Also, let's get their product name right with `macOS`. :)

```suggestion
- **macOS**: Open your Applications > Utilities folder and find "Terminal". You can also use Spotlight search to open Terminal. Press `CMD + SPACE` to open Spotlight, and search for "Terminal". Press `RETURN` to open it.
```

File: foundations/installations/command_line_basics.md (hunk 1 above)
Comment (I3uckwheat) [reply foundations/installations/command_line_basics.md:11]: Two newlines will force a new line to be displayed.

```
from
this
```

```
to

this
```

**Review by I3uckwheat — APPROVED** [review]

**Review by wise-king-sullyman — APPROVED** [review]
Looks great, thanks!

**Pull request merged**